hub_adapter/integration_test.py

```
@@ -181,20 +183,26 @@ async def cleanup_analysis(self):
+            resp = await self.http_client.delete(project_url, headers=self.token)
+            resp.raise_for_status()
+
+            ds_url = f"{self.integration_settings.api_base_url}/kong/datastore/{self.datastore_id}"
+            resp = await self.http_client.delete(ds_url, headers=self.token, params={"cascade": True})
             resp.raise_for_status()
+
             self._resources_created["datastore"] = False
             logger.info("✓ Datastore cleaned up")
 
-        except HTTPException as e:
+        except (HTTPException, httpx.HTTPStatusError) as e:
```

Ruff (0.16.0)

[error] Undefined name `httpx`

(F821)